### ISSUE TITLE
Memory leak on watching

### ISSUE THREAD

**Issue by thexpand** [simonsmith/postcss-property-lookup#26]
I'm using the plugin with sass-loader and webpack. After watching for 30 minutes or more it results in a memory leak - the node server crashes. Also, the watch compile time becomes very slow (maybe 1.2 - 1.5 times slower).

**Comment by jedmao** [comment]
Are you saying that removing the plugin gets rid of the memory leak?

**Comment by thexpand** [comment]
Yes. I have not installed or removed anything else rather than the plugin.
